### PR TITLE
SIGINT-2209: Updated Deprecated message to readme file

### PR DESCRIPTION
**Pull request by renukachalavadi**
1. Updating Readme
2. Action.yml for Banner change
3. main.ts for warning message

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {GitHubClientServiceFactory} from './synopsys-action/factory/github-clien
 
 export async function run() {
   info('Synopsys Action started...')
+  warning('This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new Black Duck Security Scan https://github.com/marketplace/actions/black-duck-security-scan. Instructions can be found at http://<community url> here')
```
Comment (zaman-akib) [inline src/main.ts:13]: We need to update the `<community url>`

**Review by lsynopsys — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# Synopsys Action
+# DEPRECATED: Synopsys Action
+**NOTE:** This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>.Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">BlackDuckDoc's</a>.
```
Comment (jahid1209) [inline README.md:2]: Should be -
This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>. Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">Black Duck Docs</a>.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # DEPRECATED: Synopsys Action
-**NOTE:** This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>.Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">BlackDuckDoc's</a>.
+**NOTE:** This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>. Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">here</a>.
```
Comment (jahid1209) [inline README.md:2]: It should be -

Instructions can be found here.

File: README.md (hunk 2 above)
Comment (renukachalavadi) [reply README.md]: I see the same text in read me file

**Review by jahid1209 — APPROVED** [review]

**Pull request merged**